### ISSUE TITLE
Test v1.5.0

### ISSUE THREAD

**Issue by canyouswim** [canyouswim/Caching-on-Kai#48]
@david-p-s - if you have time, here is v1.5.0 - latest updates in the readme file.  I did my testing this time with a non-premium account, so hopefully it should be pretty solid.

Looking forward to your thoughts!  Again - this is not urgent for today :)  
[caching-on-kai.zip](https://github.com/canyouswim/Caching-on-Kai/files/5543988/caching-on-kai.zip)

**Comment by david-p-s** [comment]
Version 1.5 is in my phone now. It seems to work. Load caches when the wifi
is turned off without problem. I have noticed that the tiles of the map are
also there even without wifi. But it still requires GPS turned on to begin.

So I think that it will seek caches in offline mode. But I haven't tried it
yet. Give me some time to find ;-)

Note:The only thing I don't like is that it hasn't been implemented my
"giveMeWaypoint" features. :-P I will program something today. I want to
put all the things that I did into reusable functions. And I will use your
new "Enter Waypoint" button, instead of *. So it can be free for show logs.
I hope it will be something to merge in master soon.

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
